src/StorageSync/StorageSync/ServerEndpoint/InvokeStorageSyncFileRecallCommand.cs

```diff
@@ -114,8 +114,8 @@ public class InvokeStorageSyncFileRecallCommand : StorageSyncClientCmdletBase
            ParameterSetName = StorageSyncParameterSets.ObjectParameterSet,
            Mandatory = true,
            ValueFromPipeline = true,
-           HelpMessage = HelpMessages.SyncGroupObjectParameter)]
-        [Alias(StorageSyncAliases.RegisteredServerAlias)]
+           HelpMessage = HelpMessages.ServerEndpointObjectParameter)]
```

Comment:
Does this alias have a new value?  If so, you should not delete it, but should add another alias

Reply:
Yes, tests are pending the SDK dependency.

Reply:
I'm fixing a bug here where InputObject was improperly described and aliased. Proper description / alias should be in ServerEndpoint-context.

If process for such fixes should be different, I can revert this change.

Reply:
That is a breakign change.  You cannot change an alias, instead, you can add an alias to a new value and deprecate the old one.

Reply:
Addressed as recommended by @markcowl